frontend/src/api/services/auth.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const url = 'https://todos-h97u.onrender.com/api/auth';
```

Comment:
Should be one general BASE_URL variable somewhere in config file to avoid duplications